service_catalog/forms/admin_forms.py

```diff
@@ -98,9 +104,55 @@ class ServiceForm(ModelForm):
                              required=False,
                              widget=forms.FileInput())
 
+    billing = forms.ChoiceField(
+        label="Billing :",
+        choices=[
+            ('defined', 'Define billing'),
+            ('User define billing', (
+                ('choice_restricted', 'Restricted billing groups'),
```

Comment:
I would change the key here for something close of the comment explanation. Like `restricted_billing_group` and `all_billing_group`